changelog_unreleased/handlebars/10958.md

```diff
@@ -0,0 +1,44 @@
+#### Preserve attributes order for element node (#10958 by @dcyriller)
+
+<!-- prettier-ignore -->
+```handlebars
+{{!-- Input --}}
+<MyComponent
+  {{! this is a comment for arg 1}}
+  @arg1="hello"
+  {{on "clik" this.modify}}
+  @arg2="hello"
+  {{! this is a comment for arg 3}}
+  @arg3="hello"
+  @arg4="hello"
+  {{! this is a comment for arg 5}}
+  @arg5="hello"
+  ...arguments
+/>
```

Comment:
Can we make this example a little shorter?

Reply:
Oops. Sorry, missed this comment, but IMHO it's good as is.

Reply:
It is odd, I thought I applied the change. If you think it's worth correcting, I'll open a PR

Reply:
It'll be a minor release without a blog post, so it's okay, don't bother